packages/react/src/components/footer/footerUtilities/FooterUtilities.test.tsx

```diff
@@ -15,4 +16,31 @@ describe('<Footer.Utilities /> spec', () => {
     const results = await axe(container);
     expect(results).toHaveNoViolations();
   });
+
+  it('should scroll to top', () => {
+    const spyScrollTo = jest.fn();
+
+    Object.defineProperty(global.window, 'scrollTo', { value: spyScrollTo });
+    Object.defineProperty(HTMLElement.prototype, 'offsetHeight', {
+      configurable: true,
+      value: 1,
+    });
+    Object.defineProperty(HTMLElement.prototype, 'offsetWidth', {
+      configurable: true,
+      value: 1,
+    });
+
+    const { container } = render(<FooterUtilities backToTopLabel="Test label" />, { wrapper: FooterWrapper });
+
+    const firstFocusable = getKeyboardFocusableElements(container)[0];
+
+    fireEvent.scroll(window, { target: { scrollY: 100 } });
+
+    const backToTopButton = container.querySelector('button[role="link"]') as HTMLButtonElement;
+
+    fireEvent.click(backToTopButton);
+
+    expect(spyScrollTo).toHaveBeenCalledWith({ top: 0 });
+    expect(firstFocusable).toHaveFocus();
+  });
```

Comment:
Oh, really nice 🎉 
Could we render a link and see if it does get the focus?
Something like this:
```tsx
const { container } = render(
  <>
    <p>Test paragraph</p>
    <a id="top-link" href="/#">
      Top link
    </a>
    <FooterWrapper>
      <FooterUtilities backToTopLabel="Test label" />
    </FooterWrapper>
  </>,
);

fireEvent.scroll(window, { target: { scrollY: 100 } });

const backToTopButton = container.querySelector('button[role="link"]') as HTMLButtonElement;

fireEvent.click(backToTopButton);

expect(spyScrollTo).toHaveBeenCalledWith({ top: 0 });
expect(container.querySelector('#top-link')).toHaveFocus();
 ```
 But this is great stuff already 👍

Reply:
Updated 👍